tools/deb/envoy_bootstrap_tmpl.json

```diff
@@ -0,0 +1,121 @@
+{
+  "stats_config": {
+    "use_all_default_tags": false
+  },
+  "admin": {
+    "access_log_path": "/dev/stdout",
+    "address": {
+      "socket_address": {
+        "address": "127.0.0.1",
+        "port_value": 15000
+      }
+    }
+  },
+  "dynamic_resources": {
+    "lds_config": {
+      "api_config_source": {
+        "api_type": "REST_LEGACY",
+        "refresh_delay": {{ .refresh_delay }},
+        "cluster_names": [
+          "rds"
+        ]
+      }
+    },
+    "cds_config": {
+      "api_config_source": {
+        "api_type": "REST_LEGACY",
+        "refresh_delay": {{ .refresh_delay }},
+        "cluster_names": [
+          "rds"
+        ]
+      }
+    },
+    "deprecated_v1": {
+      "sds_config": {
+        "api_config_source": {
+          "api_type": "REST_LEGACY",
+          "refresh_delay": {{ .refresh_delay }},
+          "cluster_names": [
+            "rds"
+          ]
+        }
+      }
+    }
+  },
+  "static_resources": {
+    "clusters": [
+      {
+        "name": "rds",
+        "type": "STRICT_DNS",
+        "connect_timeout": {{ .connect_timeout }},
+        "lb_policy": "ROUND_ROBIN",
+{{ if eq .config.ControlPlaneAuthPolicy 1 }}
+      "tls_context": {
```

Comment:
is white-spacing OK here? seems like it's off.

Reply:
Generated file seems ok.

Reply:
I think you can use one of gotmpl tricks like `-}}` to avoid whitespace skew.